src/rsz/src/RepairSetup.hh

```diff
@@ -30,6 +31,7 @@ using sta::Corner;
 using sta::dbNetwork;
 using sta::dbSta;
 using sta::DcalcAnalysisPt;
+using sta::Delay;
```

Comment:
In general I'd say try to avoid using `using` in header files. It pollutes the namespace for everyone downstream. Use the fully qualified name of things in headers (yes, it is already broken, but we should not break it more).

Reply:
I'd agree if this was outside the `rsz` namespace, but when it's inside and these are basic odb/sta types, I don't see the harm. Downstream code can step out of the namespace if pollution is an issue.

Reply:
I think it better to take this up separately as a project level discussion about how we want to handle includes/using.  There are many using statements already in this code so one more is just following the existing pattern.